Reword Preferred Crossmatch Syntax text to cover cone search too

mbtaylor opened the pull request:
Make it explicit that the DISTANCE<limit idiom should be used for cone-search-like constraints as well as sky crossmatch-like constraints.

This addresses the third point from the Ops IG RFC comments, see https://wiki.ivoa.net/twiki/bin/view/IVOA/ADQL21RFC#Operations%20Interest%20Group

mbtaylor committed:
Reword Preferred Crossmatch Syntax text to cover cone search too

Make it explicit that the DISTANCE<limit idiom should be used for
cone-search-like constraints as well as sky crossmatch-like
constraints.

This addresses the third point from the Ops IG RFC comments, see
https://wiki.ivoa.net/twiki/bin/view/IVOA/ADQL21RFC#Operations%20Interest%20Group

gmantele commented on ADQL.tex:
```diff
@@ -1647,34 +1647,31 @@ \subsubsection{Preferred crossmatch syntax}
+This section therefore recommends a preferred form of ADQL
+to use for sky crossmatching and the related cone search operation,
+namely to impose an upper limit on one of the two forms of the
+DISTANCE operator.
```
I would have rather said `DISTANCE function` instead of `DISTANCE operator`.

gmantele reviewed (dismissed):
I am ok with all these modifications. I just added one comment, but clearly not a critical one. Feel free to merge, with or without taking into account my comment.

mbtaylor committed:
Minor improvements to preferred crossmatch syntax section

Use term "function" rather than "operator" for DISTANCE;
add "sky" in section title; use "<=" as well as "<" in examples.

msdemlei reviewed (approved):
I like it.  Thanks!
